src/Header/UploadStream.h

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) 2021, 鍾淯丞, 周杰仕, 林仁鴻. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef GROUP4_HEADER_UploadStream_H_
+#define GROUP4_HEADER_UploadStream_H_
+
+#include <string>
+
+// write file to local may use a lock file or system call to signify that a certain file is currently in use.
+class UploadStream
+{
+public:
+    UploadStream();
+    // filename will include file path, if run successfully, it will return 1, else 0;
+    int writeFile(std::string filename, std::string &binary);
```

Comment:
const ref

Reply:
Please comment that this is appending to the file. Consider calling it `appendToFile`
Consider putting the filename in constructor if this class maps to a file.

Reply:
done.